index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+    <link rel="stylesheet" type="text/css" href="assets/css/style.css">
+
+    <link rel="resetsheet" type="text/reset" href="assets/css/reset.css">
+
+    <title>Game day</title>
+</head>
+<body>
+<div class="container">
+
+    <div class="row">
+
+        <div id="header">
+            <h1>   Crystal Collector!</h1>
+        </div>
+
+    </div>
+
+    <div class="row">
+
+        <div id="instructions">
+            <p> You will be give a random number at the start of the game.   </p>  
+            <p>There are four crystals below. By clicking on a crystal you will add a specific amount of points to your total score.</p>    
+            <p>You win the game by matching your total score to random number, you lose the game if your total score goes above the random number.</p>
+            <p>The value of each crystal is hidden from you until you click on it.</p>
+            <p>Each time when the game starts, the game will change the values of each crystal.</p>
+        </div>
+        
+    </div>
+
+    <div class="row">
+
+        <div class="col-sm-3">
+            <div id="scorecard1">
+                <h3 id=compScore> </h3>
+            </div>    
+        </div>
+
+        <div class="col-sm-3">
+            <div id="scorecard2">
+            <p id=win/loss>  </p> <br>
+            <p id=wins> Wins: </p> <br>
+            <p id=losses> Losses: </p> <br>
+            </div> 
+        </div>
+
+        <div class="col-sm-6">
+            <div>
```

Comment:
If you aren't using some of these <div> then you should try to get rid of them so they won't clutter your code